go/engine/identify2_test.go

```diff
@@ -71,11 +71,11 @@ func (i *Identify2WithUIDTester) MakeProofChecker(_ libkb.RemoteProofChainLink)
 	return i, nil
 }
 
-func (i *Identify2WithUIDTester) CheckHint(h libkb.SigHint) libkb.ProofError {
+func (i *Identify2WithUIDTester) CheckHint(_ *libkb.GlobalContext, h libkb.SigHint) libkb.ProofError {
```

Comment:
Is it weird that `i` has a global context member, but also receives this argument?